db/seeds.rb

```diff
@@ -6,9 +6,11 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-category1 = Category.create(name: 'Coffee')
-category2 = Category.create(name: 'Brewing')
-category3 = Category.create(name: 'Merchandise')
+category1 = Category.create(name: 'Coffee', image: "fa fa-coffee")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.